src/Sylius/Component/Resource/spec/Metadata/RegistrySpec.php

```diff
@@ -52,12 +54,24 @@ function it_throws_an_exception_if_resource_is_not_registered()
     function it_returns_specific_metadata(MetadataInterface $metadata)
     {
         $metadata->getAlias()->willReturn('app.shipping_method');
+        $metadata->getClass('model')->willReturn('ShippingMethod');
 
         $this->add($metadata);
 
         $this->get('app.shipping_method')->shouldReturn($metadata);
     }
 
+    function it_should_say_if_it_has_metadata_for_a_specific_metadata(MetadataInterface $metadata)
```

Comment:
As method is ``has($alias)`` maybe it should be named ``it_should_say_if_it_has_metadata_for_a_specific_alias``?